apps/ospTutorial/CMakeLists.txt

```diff
@@ -8,14 +8,28 @@ find_package(Boost 1.42.0 REQUIRED COMPONENTS system thread regex)
 include_directories(${Boost_INCLUDE_DIRS} ${PROJECT_SOURCE_DIR})
 link_directories(${Boost_LIBRARY_DIRS})
 
+find_package( Protobuf REQUIRED CONFIG )
+if ( Protobuf_FOUND )
+    message( STATUS "Protobuf version : ${Protobuf_VERSION}" )
+else()
+    message( WARNING "Protobuf package not found -> specify search path via Protobuf_DIR variable")
+endif()
+
 # build ospTutorial, for demonstration and API testing
 add_executable(ospTutorial ${OSPRAY_RESOURCE} ospTutorial.c)
 target_link_libraries(ospTutorial PRIVATE ospray)
 
 # C++ version
 add_executable(ospTutorialCpp ${OSPRAY_RESOURCE} ospTutorial.cpp
-                              ${PROJECT_SOURCE_DIR}/http-parser/http_parser.c)
-target_link_libraries(ospTutorialCpp PRIVATE ospray ${Boost_LIBRARIES})
+                              ${PROJECT_SOURCE_DIR}/http-parser/http_parser.c
+                              proto/FeatureCollection.pb.cc
+                              proto/FeatureCollection.pb.h)
```

Comment:
I'm pretty sure the two files should be generated with a cmake rule and not checked in.

Reply:
hmm ... we could probably do that. thought it would be easier to start if I create them once by hand, but I can look into that